src/Common/Thermodynamics/config_numerical_method.jl

```diff
@@ -82,8 +82,10 @@ function sa_numerical_method(
     _T_min::FT = T_min(param_set)
     q_pt = PhasePartition(q_tot, FT(0), q_tot) # Assume all ice
     T_2 = air_temperature(param_set, e_int, q_pt)
```

Comment:
We can give rigorous bounds here:
```suggestion
    T_2 = T_0 +
           ( e_int + q.tot * e_int_i0 )  / cv_d
```
(This is not easily evaluated from the thermodynamics functions because we assume all ice in the numerator and dry air in the denominator, to rigorously bound the max.)

Reply:
When I tried doing this (and the suggestions below) but more iterations were needed. I forget how many, my thought is that this may require a bit of experimenting/thought.

Reply:
Maybe it uses a few more iterations, because it is a rigorous bound (i.e., maximally wide). But if the point of regula falsi is to bracket a guaranteed solution, I think we have to use rigorous brackets. What's currently there does not always bracket the zero.

Reply:
Actually, this fails even on the reference profiles (it's somehow not bracketing).
```julia
ERROR: LoadError: AssertionError: y0 * y1 < 0
Stacktrace:
 [1] find_zero(::ClimateMachine.Thermodynamics.var"#2#3"{EarthParameterSet,Float64,Float64,Float64,UnionAll}, ::RegulaFalsiMethod{Float64}, ::Float64, ::Float64, ::CompactSolution, ::ResidualTolerance{Float64}, ::Int64) at /Users/charliekawczynski/.julia/packages/RootSolvers/3Wfce/src/RootSolvers.jl:338
```

Reply:
I'm fine with skipping the RegulaFalsi changes for now, and just updating the plots to include the reference profiles. Does this sound okay @tapios?

Reply:
Why not including the rigorous bounds in RF? I don't want these to get lost (I didn't write down the derivation anywhere else.)